### DISCUSSION TITLE
Is it possible to use MistoLine model in Fooocus?

### DISCUSSION THREAD

**Discussion by thiner** [lllyasviel/Fooocus#2917] [Ideas]
This new model seems excellent on rendering sketch (canny). The model page: https://huggingface.co/TheMistoAI/MistoLine

**Comment by mashb1t** [comment]
It sure seems to be good! Generally speaking it's possible to use other ControlNets, but every preprocessor / model has to be manually integrated into the code. I currently do not the capacity to also handle ControlNets, but it's in the backlog.

**Comment by mashb1t** [comment]
also see https://github.com/lllyasviel/Fooocus/discussions/2894
